requestor/README.md

```diff
@@ -51,8 +51,103 @@ Default name of this subnet is `erigon`.
 
 ## Use requestor server
 
+### Endpoints
+
 All requests must contain a header `{'Authorization': 'Bearer [ETHEREUM_ADDRESS]'}`
 
     GET  /getInstances      - list of all instances created by the user (includes stopped instances)
     POST /createInstance    - create new instance, this requires "params" in body and also accepts optional "name"
     POST /stopInstance/<id> - stop instance with id <id>
+
+### Command-line interface
+
+All further examples assume having a Python environment set up and active. It needs to be done once before using the CLI.
+```shell
+python3 -m venv .venv
+. .venv/bin/activate
```

Comment:
It was OK - but maybe this is more straight 🤔 Leaving to you
```suggestion
source .venv/bin/activate
```

Reply:
> `source` is a synonym for dot/period '.' in bash, but not in POSIX sh, so for maximum compatibility use the period.

([source](https://ss64.com/bash/source.html))